Userland/Libraries/LibWeb/DOM/Node.cpp

```diff
@@ -2312,11 +2314,18 @@ ErrorOr<String> Node::name_or_description(NameOrDescription target, Document con
             // - Otherwise, return the value of aria-label.
             return element->aria_label().value();
         }
-        // TODO: D. Otherwise, if the current node's native markup provides an attribute (e.g. title) or element (e.g. HTML label) that defines a text alternative,
-        //      return that alternative in the form of a flat string as defined by the host language, unless the element is marked as presentational (role="presentation" or role="none").
+        // E. Host Language Label: Otherwise, if the current node's native markup provides
```

Comment:
Let's add newlines between code an spec steps please (i.e. after the previous closing `}`). And in the rest of the code base, we typically wrap spec comment lines at about 120 columns.